### PR TITLE
Added Dockerfile for release/cepp

### PR DESCRIPTION
**Pull request by izderic**
* Added Dockerfile for release/cepp
* Moved `dargs`, `sequelize-cli` and `umzug` to regular dependencies to fix Module not Found errors
* Installed missing `mississippi` package

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.git
+.env
```
Comment (ikovac) [inline .dockerignore:3]: We can also ignore the dist folder?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM node:14.21.3-slim@sha256:21b6dc452bd2c066724bd5be0f7256ef81a068c47de1adead10a2f810557de02 AS base
```
Comment (ikovac) [inline Dockerfile:1]: Why are we not using bookworm-slim for the runner image?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+RUN apt update && apt install -y --no-install-recommends dumb-init
+ENTRYPOINT ["dumb-init", "--"]
+
+FROM node:14.21.3@sha256:2cafa3fbb0b6529ee4726b4f599ec27ee557ea3dea7019182323b3779959927f AS install
```
Comment (ikovac) [inline Dockerfile:5]: We can use bookworm image as install image

File: package.json [hunk 4]
```diff
@@ -122,6 +123,7 @@
     "pg-query-stream": "^3.2.0",
     "pluralize": "^8.0.0",
     "plyrue": "^2.1.4",
+    "poi": "^12.10.3",
```
Comment (ikovac) [inline package.json]: Please revert this change and install missing `mississipi` package (required in amazon storage file)

**Review by ikovac — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (izderic) [reply Dockerfile:1]: I could not find bookworm-slim for this node version.

File: Dockerfile (hunk 3 above)
Comment (izderic) [reply Dockerfile:5]: Looks like bookworm and bookworm-slim are available only in newer versions of node.

**Review by ikovac — APPROVED** [review]

**Review by abasic — APPROVED** [review]

**Pull request merged**